### DISCUSSION TITLE
Problems with installation in Azure - certbot and letsencrypt

### DISCUSSION THREAD

**Discussion by isbarrerag** [mediacms-io/mediacms#720] [General]
Hello everyone, I have problemsproblems with installation on Azure - certbot and letsencrypt.
Server: Ubuntu 20.04.5, RAM:16,vCPU 4
I comment that it does the installation in single, port 80 and 443 work.
in https it throws the following errors:
"IMPORTANT NOTES:
  - The following errors were reported by the server:
    Domain: server.eastus.cloudapp.azure.com
    Type: connection
    Detail: XX.XXX.XXX.XXX: Fetching
    http://server.eastus.cloudapp.azure.com/.well-known/acme-challenge/NMQGCSb0dgbXG8y7g770-fcwiOvSW0fH2W2eQveaQuI:
    Timeout during connect (likely firewall problem)

    To fix these errors, please make sure that your domain name was
    entered correctly and the DNS A/AAAA record(s) for that domain
    contain(s) the right IP address. Additionally, please check that
    your computer has a publicly routable IP address and that does not
    firewalls are preventing the server from communicating with the
    client. If you're using the webroot plugin, you should also verify
    that you are serving files from the webroot path you provided."

when I run the command:
#certbot --nginx -n --agree-tos --register-unsafely-without-email -d server.eastus.cloudapp.azure.com
ocurrs the same...

I check the firewall and find no problem.
fwd off.
The certificate also shows expired...
Issued on Tuesday, March 10, 2020, 2:51:41 PM
Due on Monday, June 8, 2020, 13:51:41

I have looked for several ways to solve it and they don't work... any ideas? some specific configuration in azure?
Regards

**Comment by mgogoulos** [comment]
Hi, what are you using, the server install or the docker install? Not sure at what point you are getting this error, can you provide the steps you tried, and also the exact output?
